packages/web-components/scripts/verify-packaging.js

```diff
@@ -0,0 +1,77 @@
+// @ts-check
+/**
+ * Copied from ${@link 'file://./../../../scripts/tasks/src/verify-packaging.ts'}
```

Comment:
following the pattern - just-scripts doesn't support ESM, we will migrate away from just-scripts anyways so copy-paste for now ( with tweaks )